did/resolver.go

```diff
@@ -1,20 +1,21 @@
 package did
 
 import (
+	"context"
 	"fmt"
 	"strings"
 
 	"github.com/goccy/go-json"
 	"github.com/pkg/errors"
 )
 
-// ResolutionOptions https://www.w3.org/TR/did-spec-registries/#did-resolution-options
-type ResolutionOptions any
+// ResolutionOption https://www.w3.org/TR/did-spec-registries/#did-resolution-options
+type ResolutionOption any
```

Comment:
Whenever I see `any`, it's typically a symptom of an API that hasn't been fleshed out yet. In this case, it's unclear the shape of what `ResolutionOption` is acceptable by the functions that use it. 

Instead, I would define an interface. Mind creating an issue to address?

Separately, This doesn't seem to be used anywhere.

Reply:
yeah not used anywhere yet...will create an issue

Reply:
https://github.com/TBD54566975/ssi-sdk/issues/331